webrender/src/frame_builder.rs

```diff
@@ -500,6 +502,7 @@ impl FrameBuilder {
             frame_rect,
             content_size,
             scroll_sensitivity,
+            enable_scrollbars
```

Comment:
I'm not sure we need to pass this value to the ClipScrollNode. I think you can just add the scrollbar here as a child of the parent ClipScrollNode. You might need to do some special calculation to ensure that the scrollbar stays within the rectangle of the parent node.